homeassistant/core.py

```diff
@@ -165,9 +161,10 @@ def async_start(self):
 
         # pylint: disable=protected-access
         self.loop._thread_ident = threading.get_ident()
-        _async_create_timer(self)
         self.bus.async_fire(EVENT_HOMEASSISTANT_START)
+        yield from self.async_stop_track_tasks()
```

Comment:
Maybe add a timeout to `async_stop_track_tasks` after which we will no longer stop tracking tasks?